packages/example/src/components/remote-control/OverridableEventEmitter.ts

```diff
@@ -0,0 +1,25 @@
+import { SupportedKeys } from './SupportedKeys';
+
+class OverridableEventEmitter {
+  private handlers = new Map<string, ((event: SupportedKeys) => boolean)[]>();
```

Comment:
Ideally, we'd like to type like mitt does right now. Mitt allows to type what are the events and for each event what is the param signature. You should have a look at how mitt does it :)

```suggestion
  private handlers = new Map<T, U[]>();
```

Reply:
Yeah, I saw how they handled it in mitt but I figured we didn't need the whole implementation for our use case :-) Can we take a look at it together quickly so we can agree on what to and what not to implement?

Reply:
Done 👍🏽